nixos/modules/system/boot/loader/systemd-boot/systemd-boot-builder.py

```diff
@@ -197,6 +197,22 @@ def main():
             subprocess.check_call(["@systemd@/bin/bootctl", "--path=@efiSysMountPoint@", "install"])
         else:
             subprocess.check_call(["@systemd@/bin/bootctl", "--path=@efiSysMountPoint@", "--no-variables", "install"])
+    else:
+        # Update bootloader to latest if needed
+        systemd_version = subprocess.check_output(["@systemd@/bin/bootctl", "--version"], universal_newlines=True).split()[1]
+        sdboot_status = subprocess.check_output(["@systemd@/bin/bootctl", "--path=@efiSysMountPoint@", "status"], universal_newlines=True)
```

Comment:
This line fails on my system, because of `Boot into FW: not supported`, which causes the entire command to fail, when then causes `check_output` to throw an exception. Is that the expected behavior in this case?

Reply:
Ignore this comment, I submitted an issue about this instead.